readthedocs/projects/tasks.py

```diff
@@ -1322,27 +1322,19 @@ def update_static_metadata(project_pk, path=None):
 
 # Random Tasks
 @app.task()
-def remove_dir(path):
+def remove_dirs(paths):
     """
-    Remove a directory on the build/celery server.
+    Remove artifacts from the web servers.
```

Comment:
Just in case, it's not "from the web servers". It will depends on how it's triggered/broadcasted.

In the code there are places where we are doing `broadcast(type='builds'...)` for example.

Reply:
Yep, this should be fixed ^^